--- /dev/null
+++ b/ui/gtk/native_theme_gtk.cc
@@ -497,6 +497,16 @@
   set_high_contrast(theme_name.find("high") != std::string::npos &&
                     theme_name.find("contrast") != std::string::npos);
 
+  // Brute force NativeUI to update
+  ui::NativeTheme::GetInstanceForNativeUi()->set_use_dark_colors(color_utils::IsDark(GetSystemColor(kColorId_WindowBackground)));
+  ui::NativeTheme::GetInstanceForNativeUi()->set_preferred_color_scheme(CalculatePreferredColorScheme());
+  ui::NativeTheme::GetInstanceForNativeUi()->NotifyObservers();
+
+  // Brute force Web to update
+  ui::NativeTheme::GetInstanceForWeb()->set_use_dark_colors(color_utils::IsDark(GetSystemColor(kColorId_WindowBackground)));
+  ui::NativeTheme::GetInstanceForWeb()->set_preferred_color_scheme(CalculatePreferredColorScheme());
+  ui::NativeTheme::GetInstanceForWeb()->NotifyObservers();
+  
   NotifyObservers();
 }
 
