diff --git a/PKGBUILD b/PKGBUILD
index 73d56d8..531cc0f 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -33,14 +33,18 @@ source=(https://commondatastorage.googleapis.com/chromium-browser-official/chrom
         webcodecs-stop-using-AudioOpusEncoder.patch
         webrtc-check-existence-of-cursor-metadata.patch
         sql-make-VirtualCursor-standard-layout-type.patch
-        use-oauth2-client-switches-as-default.patch)
+        use-oauth2-client-switches-as-default.patch
+        https://raw.githubusercontent.com/qcasey/chromium-patches/main/patches/custom-startpage.patch
+        https://raw.githubusercontent.com/qcasey/chromium-patches/main/patches/gtk-dark-mode-switch-fix.patch)
 sha256sums=('0e5ea5f3061ad090cf6bd57ca037496d95ea8956de021aff902f7d0ded7bffdc'
             '213e50f48b67feb4441078d50b0fd431df34323be15be97c55302d3fdac4483a'
             'a6120e7d4eb5e131b87b6ab3b922e0c6cd78e15501e54cfb2019875173688d80'
             '064daaa2b9d95b96ec04d8ddebf4af441f92263d123365b58fe73966866080af'
             '88b2c8d9c6c1917f6632453f18aad7a3fd94d605eecb6c77ae2394ac5856ba95'
             'b94b2e88f63cfb7087486508b8139599c89f96d7a4181c61fec4b4e250ca327a'
-            'e393174d7695d0bafed69e868c5fbfecf07aa6969f3b64596d0bae8b067e1711')
+            'e393174d7695d0bafed69e868c5fbfecf07aa6969f3b64596d0bae8b067e1711'
+            '11f4222cf889edb01ae6eabd5da7362dccfed38ce0f59c12a58cf152f35ea6bb'
+            'bed68de954f6a0174db3e1caf2e3ae173384984de4808672f8eecf0196103031')
 provides=('chromium')
 conflicts=('chromium')
 source=(${source[@]}
@@ -92,6 +96,12 @@ prepare() {
     third_party/libxml/chromium/*.cc \
     third_party/maldoca/src/maldoca/ole/oss_utils.h
 
+  # Custom startpage with correct tab focus
+  patch -Np1 -i ../custom-startpage.patch
+
+  # Fix GTK Dark Mode theme switching (or brutally brute force it)
+  patch -Np1 -i ../gtk-dark-mode-switch-fix.patch
+
   # Use the --oauth2-client-id= and --oauth2-client-secret= switches for
   # setting GOOGLE_DEFAULT_CLIENT_ID and GOOGLE_DEFAULT_CLIENT_SECRET at
   # runtime -- this allows signing into Chromium without baked-in values