diff --git a/PKGBUILD b/PKGBUILD
index ca8a1ff..b0b0945 100644
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -35,7 +35,9 @@ source=(https://commondatastorage.googleapis.com/chromium-browser-official/chrom
         REVERT-roll-src-third_party-ffmpeg-m102.patch
         REVERT-roll-src-third_party-ffmpeg-m106.patch
         angle-wayland-include-protocol.patch
-        use-oauth2-client-switches-as-default.patch)
+        use-oauth2-client-switches-as-default.patch
+        https://raw.githubusercontent.com/qcasey/chromium-patches/main/patches/custom-startpage.patch
+        https://raw.githubusercontent.com/qcasey/chromium-patches/main/patches/gtk-dark-mode-switch-fix.patch)
 sha256sums=('0f091b4950e120d5c3e23ab391bacfdb6ce8eb8d0acab55e9dae1a8c247dd192'
             '213e50f48b67feb4441078d50b0fd431df34323be15be97c55302d3fdac4483a'
             '2ad419439379d17385b7fd99039aca875ba36ca31b591b9cd4ccef84273be121'
@@ -44,7 +46,9 @@ sha256sums=('0f091b4950e120d5c3e23ab391bacfdb6ce8eb8d0acab55e9dae1a8c247dd192'
             '30df59a9e2d95dcb720357ec4a83d9be51e59cc5551365da4c0073e68ccdec44'
             '4c12d31d020799d31355faa7d1fe2a5a807f7458e7f0c374adf55edb37032152'
             'cd0d9d2a1d6a522d47c3c0891dabe4ad72eabbebc0fe5642b9e22efa3d5ee572'
-            'e393174d7695d0bafed69e868c5fbfecf07aa6969f3b64596d0bae8b067e1711')
+            'e393174d7695d0bafed69e868c5fbfecf07aa6969f3b64596d0bae8b067e1711'
+            'ce3ca435bd8cde5708595f89ce65f544e312adb5835a4aa3a4ebc8cd7bdf11cc'
+            '07134c33e3947b8ef6f070f732d79e7998db2d3271b5f29bb28ce06748a5ad79')

 provides=('chromium')
 conflicts=('chromium')
@@ -149,6 +153,12 @@ prepare() {
   python "$_utils/domain_substitution.py" apply -r "$_ungoogled_repo/domain_regex.list" \
     -f "$_ungoogled_repo/domain_substitution.list" -c domainsubcache.tar.gz ./
 
+  # Custom startpage with correct tab focus
+  patch -Np1 -i ../custom-startpage.patch
+
+  # Fix GTK Dark Mode theme switching (or brutally brute force it)
+  patch -Np1 -i ../gtk-dark-mode-switch-fix.patch
+
   # Link to system tools required by the build
   mkdir -p third_party/node/linux/node-linux-x64/bin
   ln -s /usr/bin/node third_party/node/linux/node-linux-x64/bin/
